### PR TITLE
COMMERCE-11742 Add support for UoM to CPDefinitionOptionValueRel

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/COMMERCE-11742

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/action/EditCPDefinitionOptionValueRelMVCActionCommand.java [hunk 1]
```diff
@@ -151,7 +152,7 @@ private CPDefinitionOptionValueRel _updateCPDefinitionOptionValueRel(
 
 			return _cpDefinitionOptionValueRelService.
 				addCPDefinitionOptionValueRel(
-					cpDefinitionOptionRelId, nameMap, priority, key,
+					cpDefinitionOptionRelId, key, nameMap, priority,
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/action/EditCPDefinitionOptionValueRelMVCActionCommand.java:155]: sort also the parameters above this method call

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
check the sort comment

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:commerce-functional

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
resent here https://github.com/liferay-commerce/liferay-portal/pull/4087

**Pull request closed without merging**